python2/ltk/actions/init_action.py

```diff
@@ -171,9 +171,9 @@ def init_action(self, host, access_token, client_id, project_path, folder_name,
                     self.config_parser.set('main', 'project_name', project_name)
 
                 # get workflow
-                logger.info('---------------------------')
-                logger.info('SELECT TRANSLATION WORKFLOW')
-                logger.info('---------------------------')
+                logger.info('----------------------------------------------')
+                logger.info('SELECT PROJECT DEFAULT TRANSLATION WORKFLOW')
+                logger.info('----------------------------------------------')
```

Comment:
This can be a little misleading since the user may think that by selecting a workflow here, the FSC would update the workflow used by the project.

Reply:
From what I understand that is what's happening with this option. The workflow chosen here is set to be the new `Project Default` workflow. If the original `Project Default` is chosen, then nothing happens.